integration-tests/spring-extension-support-test/src/test/java/com/example/application/AutoCrudTest.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2023 Marco Collovati, Dario Götze
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.application;
+
+import java.util.List;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+import io.quarkus.test.TestTransaction;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
+import com.github.mcollovati.quarkus.testing.AbstractTest;
+import com.github.mcollovati.quarkus.testing.VaadinConditions;
+
+import static com.codeborne.selenide.CollectionCondition.size;
+import static com.codeborne.selenide.Selectors.byXpath;
+import static com.codeborne.selenide.Selectors.shadowDeepCss;
+import static com.codeborne.selenide.Selenide.$;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@QuarkusTest
+@TestTransaction
+class AutoCrudTest extends AbstractTest {
+
+    @Override
+    protected String getTestUrl() {
+        return getBaseURL() + "auto-crud";
+    }
+
+    @Test
+    void autocrud_gridAndFormDisplayed() {
+        openAndWait(() -> $("div.auto-crud"));
+
+        SelenideElement grid =
+                $("div.auto-crud div.auto-crud-main > vaadin-grid").shouldBe(Condition.visible);
+
+        assertThat(collectColumnTexts(grid, 1, TestData.RENDERED_ITEMS)).containsExactlyElementsOf(TestData.NAMES_ASC);
+
+        SelenideElement form = $("div.auto-crud div.auto-form").shouldBe(Condition.visible);
+        form.$("vaadin-text-field[name=name]").shouldBe(VaadinConditions.disabled);
+        form.$("vaadin-text-field[name=surname]").shouldBe(VaadinConditions.disabled);
+        form.$("div.auto-form-toolbar vaadin-button[theme=primary]").shouldBe(VaadinConditions.disabled);
+    }
+
+    @Test
+    void autocrud_newUser_userSaved() {
+        openAndWait(() -> $("div.auto-crud"));
+
+        SelenideElement newButton =
+                $("div.auto-crud-toolbar vaadin-button[theme=primary]").shouldNotBe(VaadinConditions.disabled);
+        newButton.click();
```

Comment:
Same here

Reply:
As before, I prefer assign + call for clarity

Reply:
Also apply it to the other button

Reply:
refactored tests